common/Alexa-control-the-Bluetooth-Mesh-devices-via-cloud.md

```diff
@@ -0,0 +1,1031 @@
+The developer can choose either typing or using JSON formatted directive to test the Alexa skill. It simulates a real echo plus speaker to send the directives to the server where holding the Alexa skill.
+For example, if the user type or say “turn off light”, the Alexa server will send a turn-off directive to the lambda(the skill holder server).
+
+<div align="center">
+  <img src="./files/CM-Smart-Speaker/skill_test.png">
+  <center> <b>Figure: Skill-testing</b> </center>
+</div>  
+
+###	Add a Lambda Function
+#### Create a lambda function
+The code for your smart home skill is hosted as a Lambda function on AWS. AWS Lambda is a service that lets you run code in the cloud without managing servers. Alexa sends your skill requests and your code inspects the request, takes any necessary actions such as communicating with the device cloud for that customer, and then sends back a response.
+
+1.	Go to https://console.aws.amazon.com/console/home and sign in
+2.	Switch the AWS server to “US East (N.Virginia)” on the top right of the page, because some of the features are only supported on this server. Below is the recommended service for different regions.
+* N.Virginia for English (US) or English (CA) skills
+* EU (Ireland) region for English (UK), English (IN), German or French (FR) skills
+* US West (Oregon) for Japanese and English (AU) skills.
+
+3.	Go to Services > Compute > Lambda
+4.	Click on Create Function
+5.	Click on “Author from scratch”
+6.	Configure your Lambda function
+* Name = alexaBtmeshBridgeLambda (or whatever you want)
+* Runtime = Python 3.6 (You can choose any language used to write your function, in this demo, we are using Python 3.6)
+* Permission = “Create a new role with basic Lambda permissions”
+* Click Create Function
+
+<div align="center">
+  <img src="./files/CM-Smart-Speaker/create_a_lambda_function.gif">
+</div>  
+</br>
+
+7.	Click Triggers -> Add Trigger and select **Alexa Smart Home**
+* Application Id = <a href="#YourSkillID" target="_self">Skill ID</a> of your test skill that your created above.
+* Enable trigger = checked
+* Click Submit
+
+<div align="center">
+  <img src="./files/CM-Smart-Speaker/add_trigger_for_lambda_function.gif">
+</div>  
+</br>
+
+8.	Click the icon of the lambda function for Configuration
+
+<div align="center">
+  <img src="./files/CM-Smart-Speaker/lambda_function_configuration.png">
+</div>  
+</br>
+
+Regarding to the lambda function, we take the *Alexa Smart Home skill sample* released by Amazon as starting point, you can access it from the repository [skill-sample-python-smarthome-switch](https://github.com/alexa/skill-sample-python-smarthome-switch). 
+And also you can just reuse the lambda function we have implemented for this project, the zip package ***alexa-control-btmesh-lambda.zip*** is included in the sub directory and also you can get it from the [github](https://github.com/ChengYuan-CY/Alexa-Control-Bluetooth-Mesh-Devices/tree/master/lambda_function).
```

Comment:
Please upload the alexa-control-btmesh-lambda.zip to the master branch, and modify the GitHub link as the correct one.